### DISCUSSION TITLE
duda warning

### DISCUSSION THREAD

**Discussion by rominaarancibia** [IIC2133-PUC/2024-1#122] [Tarea 1] [answered]
Hola! Esta warning es problemática o da lo mismo?

<img width="911" alt="Captura de pantalla 2024-04-24 a la(s) 4 57 54 p  m" src="https://github.com/IIC2133-PUC/2024-1/assets/127449220/ebbf39c9-8877-4941-b6e2-02b840eff2fa">

**Comment by benjavicente** [comment, marked as answer]
No. `-lm`es una opción que permite que `math.h` esté disponible. Si no usan `math.h` les saldrá eso, pero no hay problema.

[Hay buenas explicaciones en SO](https://stackoverflow.com/a/64090893/19317456).
